File: Results/DER/Translation/translation_output/no_test/granite/codenet/java/python/p02054.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```:
```
import java.util.Scanner;

public class p02054 {
	static Scanner sc = new Scanner(System.in);

	public static void main(String[] args) {
		int odd = 0;
		for (int i = 0; i < 3; i++) {
			odd += sc.nextInt() % 2;
		}
		System.out.println(odd >= 2? "Hom" : "Tem");
	}
}


```


Answer:
```python
import sys

odd = 0
for i in range(3):
    odd += int(sys.stdin.readline()) % 2

print("Hom" if odd >= 2 else "Tem")
```<|endoftext|>

File: Results/DER/Translation/translation_output/w_test/granite/codenet/python/java-sanitized/p02054.java
import java.util.*;

public class p02054 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        if ((a * b) % 2 == 0 && (b * c) % 2 == 0 && (c * a) % 2 == 0) {
            System.out.println("Tem");
        } else {
            System.out.println("Hom");
        }
    }
}